what4/src/What4/Protocol/SMTWriter.hs

```diff
@@ -726,6 +735,12 @@ newWriterConn h in_h ack solver_name features bindings cs = do
                        , consumeAcknowledgement = ack
                        }
 
+-- | Strictness level for parsing solver responses
```

Comment:
Probably would be nice to have a comment on this datatype describing the different modes.